hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -58,6 +58,8 @@ public class OMMetrics implements OmMetadataReaderMetrics {
   private @Metric MutableCounterLong numKeyLookup;
   private @Metric MutableCounterLong numKeyRenames;
   private @Metric MutableCounterLong numKeyDeletes;
+  private @Metric MutableCounterLong numKeyRetentionDeletes;
```

Comment:
Please add a comment for numKeyDeletes, that numKeyDeletes counts only delete directly from user.

Reply:
This does not modify the original meaning of numKeyDeletes, which is that all deletions are counted.